nix/tests/expected/storage.out

```diff
@@ -102,34 +102,6 @@ order by
  storage     | buckets    | supabase_storage_admin | TRIGGER        | f
  storage     | buckets    | supabase_storage_admin | TRUNCATE       | f
  storage     | buckets    | supabase_storage_admin | UPDATE         | f
- storage     | migrations | anon                   | DELETE         | f
- storage     | migrations | anon                   | INSERT         | f
```

Comment:
Quite surprising to see the API roles having privileges over storage internal tables.